changes/3068.feature.rst

```diff
@@ -0,0 +1 @@
+Added support for cookie retrieval from a webview
```

Comment:
The change note should be phrased in the way you'd expect the "great new feature" to be described in the next release notes:
```suggestion
Webviews now support the retrieval of cookies.
```